src/app/resources/view-resources/resources-view.component.html

```diff
@@ -11,6 +11,7 @@
     </button>
     <span class="toolbar-fill"></span>
     <ng-container *ngIf="!parent">
+      <a mat-raised-button [href]="resourceSrc" target="_blank" color="accent">Open in new tab</a>

```

Comment:
add i18n here.
      <a mat-raised-button [href]="resourceSrc" target="_blank" color="accent" i18n>Open in new tab</a>